### DISCUSSION TITLE
Which feed does birdseye display, the main or the sub, the record or detect?

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#10162] [General]
Birdseye dimensions can be specified to "increase the quality". Which feed does birdseye use to display, the sub or the main? And does it stand to reason then that the specified dimensions should never exceed that feed?

**Comment by NickM-27** [comment]
Birdseye uses the detect streams because it must use decoded streams. And no it doesn't set a limit because either feed can be resized to fit the other

**Reply by thinkloop** [reply]
Thank you. What do you mean either feed can be resized to fit the other? If my detect stream is 800x600 but I specify 1920x1080 for birdseye, will I actually get better quality than 800x600?

**Reply by NickM-27** [reply]
The 800x600 image will be scaled up to fit inside the 1920x1080

**Reply by thinkloop** [reply]
Ok. Can we agree that there is no purpose to doing that since there isn't actually more information/quality to display and since the size/scale of the image does not change?

**Reply by NickM-27** [reply]
The purpose is that the image fits the canvas and doesn't have black bars around it.
